test/Microsoft.Azure.EventHubs.Processor.UnitTests/TestEventProcessor.cs

```diff
@@ -22,13 +22,13 @@ public TestEventProcessor()
         Task IEventProcessor.CloseAsync(PartitionContext context, CloseReason reason)
         {
             this.OnClose?.Invoke(this, new Tuple<PartitionContext, CloseReason>(context, reason));
-            return Task.CompletedTask;
+            return Task.FromResult<object>(null);
```

Comment:
It seems backwards to go from Task.CompletedTask, but I dont think this is available in .NET 451